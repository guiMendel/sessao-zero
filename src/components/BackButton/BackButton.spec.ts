import { mountOptions } from '@/tests'
import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import { BackButton } from '.'

describe('BackButton', () => {
  it('should render properly', () => {
    const wrapper = mount(BackButton, mountOptions)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
