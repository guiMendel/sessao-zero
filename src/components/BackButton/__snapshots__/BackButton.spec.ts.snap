// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`BackButton > should render properly 1`] = `
"<div data-v-60e86206=\\"\\" class=\\"back-button\\">
  <font-awesome-icon-stub data-v-520ca059=\\"\\" data-v-60e86206=\\"\\" class=\\"button-icon\\" icon=\\"fas,hand-point-left\\"></font-awesome-icon-stub>
</div>"
`;
